### DISCUSSION TITLE
Build Error

### DISCUSSION THREAD

**Discussion by ShawnCentra** [vercel/community#511] [Help] [answered]
```
Failed to compile.
./pages/careers/current-openings/[name].js
Module not found: Can't resolve './jobDetail.module.scss' in '/vercel/path0/centra-website/pages/careers/current-openings'
./pages/windows/performance.js
Module not found: Can't resolve './Performance.module.scss' in '/vercel/path0/centra-website/pages/windows'
> Build failed because of webpack errors
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Command "yarn run build" exited with 1
```

Getting some import errors where when i check the file in question, the import looks correct. What else could be the issue? I'm guessing when i swapped some imports around by moving the file in VSC, the autofix import paths causes this issue? unsure. Any ideas?

**Comment by swarnava** [comment, marked as answer]
Are you able to reproduce this locally?

**Reply by ShawnCentra** [reply]
nope, it builds just fine on local and im able to next run start it just fine.

**Reply by swarnava** [reply]
We might need to take a look at the code. Could you create a case with steps to reproduce this using https://vercel.com/support/request ?
